src/components/NavItems/tools/Thumbnails/Thumbnails.jsx

```diff
@@ -158,7 +158,7 @@ const Thumbnails = () => {
         }),
       );
       if (selectedValue.openTabs) images.forEach((img) => imageClickUrl(img));
-    } else dispatch(setError("Please use a valid Youtube Url (add to tsv)"));
+    } else dispatch(setError("Please use a valid YouTube URL (add to tsv)"));
```

Comment:
This will be shown to the end user, so could you provide a tsv keyword instead?

Reply:
Please can you add me to https://github.com/AFP-Medialab/InVID-Translations so that I can push it up?

Reply:
Done ✅

Reply:
Thanks. Change made and PR submitted